app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,18 @@
+module SessionsHelper
+  def log_in user
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    @current_user ||= User.find_by id: session[:user_id]
```

Comment:
Theo em nghĩ toán tử này sẽ thực hiện công việc như sau:
+ Nếu vế trái là nil hoặc false thì VT = VP
+ Nếu vế trái khác nil hoặc false thì nó bằng chính nó
=> Nếu người dùng đã đang đăng nhập thì không được gán bằng người khác